.github/workflows/website.yml

```diff
@@ -0,0 +1,21 @@
+name: website
+
+on:
+  pull_request:
+    branches:
+      - master
```

Comment:
As before: updating the site every pull_request event that changes docs/ isn't too much?
the push event seems enough for me